README.md

```diff
@@ -1,14 +1,33 @@
 ## react-native-sample-apps
 
-## Requirements
+## Authenticating with the package repository (Android)
 
-* react: 18.2.0
-* react-native: 0.72.3
-* Android: 22+
-* iOS: 11.0+
-* Verint-XM React Native SDK 3.0.1+
+You will need to authenticate with GitHub Packages to download our native Android library.
+To do so, you'll need a personal key which can be generated from your GitHub account by following the [instructions](https://docs.github.com/en/authentication/keeping-your-account-and-data-secure/managing-your-personal-access-tokens#creating-a-fine-grained-personal-access-token). 
+The token will need the `read:packages` permission.
+
+Once you have that key, you should set two environment variables on your machine: `GITHUB_USERNAME` for your username, and `GITHUB_PERSONAL_KEY` for your personal key.
```

Comment:
Did you test this out successfully? I've found this doesn't work reliably in our SDK project - I think you should use the same method as used in the samples (`local.properties` file).